coral/config.json

```diff
@@ -26,321 +26,200 @@
   "api-30": {
     "naked": {
       "overlays-dir": "",
-      "product-bytecode": [
-        "product/app/QtiTelephonyService/QtiTelephonyService.apk",
-        "product/app/datastatusnotification/datastatusnotification.apk",
-        "product/app/uceShimService/uceShimService.apk",
-        "product/priv-app/ims/ims.apk",
```

Comment:
This is now in system_ext

https://git.rip/dumps/google/flame/-/blob/flame-user-11-RP1A.200720.009-6720564-release-keys/all_files.txt